src/foundry/client/data/documents/combatant.d.mts

```diff
@@ -434,6 +435,9 @@ declare global {
       type Get<Scope extends Flags.Scope, Key extends Flags.Key<Scope>> = Document.GetFlag<Name, Scope, Key>;
     }
 
+    // TODO: figure out why boolean is supposedly allowed here
```

Comment:
Is this for a follow up PR? It may just be someone being over-zealously allowing something with a reasonable toString implementation.